net-misc/ntp/ntp-4.2.8p11.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A network protocol suite"
+DESCRIPTION="network time protocol suite/programs"
+HOMEPAGE="http://www.ntp.org"
```

Comment:
Suggestion, to have a 100% valid URI:
```diff
-HOMEPAGE="http://www.ntp.org"
+HOMEPAGE="http://www.ntp.org/"
```
BTW, for the `DESCRIPTION`, here's a suggestion which might be prefered over `network time protocol suite/programs`:
```
DESCRIPTION="Network Time Protocol, or NTP, is a protocol designed to \
synchronize the clocks of computers over a network. The command line programs \
provided in this package are the implementation by the Network Time Foundation."
```
Well, I'm not really satisfied with my suggestion, so please feel free to keep your description.
(Maybe someone will find a better description.)

Reply:
Updated with the suggestions :)